### ISSUE TITLE
accessible deployment of napari-iohub

### ISSUE THREAD

**Issue by mattersoflight** [czbiohub-sf/napari-iohub#17]
In discussion with our potential users, we have realized that the installation of `napari-iohub` is a barrier. I suggest simplifying access as follows.
* Build a docker image with `napari-iohub`, `napari-animation`, and few other utilities important for our analysis workflow. 
* Make the docker image available on our HPC such that it can be launched from CLI just like how we launch micro-manager. The workflow can look like:
       * `module load comp.micro`, `module load dataviz`.
       * `micro-manager` and/or `napari`.

**Comment by ziw-liu** [comment]
For Micro-Manager we needed the containerization because the Java/C++ project does not provide Linux binaries and there's only build instructions for Ubuntu (while Bruno runs Rocky), so we are using Apptainer to run Ubuntu on top of Rocky.

Meanwhile, napari-iohub is a pure-Python package and does not have the same constraints. @JoOkuma maintains a shared Conda environment (`improc`) that contains most of `napari-iohub`'s dependency (and all of the required ones) and has been used by multiple internal users for similar purposes (not having to install napari and plugins themselves). I think we should either consolidate effort or at least try to replicate that simpler solution first, as setting up the Conda environment needs to be done regardless of the choice about containerization.

**Comment by mattersoflight** [comment]
@ziw-liu makes sense to maintain a shared conda environment, but there are two caveats:
* only one user id can maintain a given environment. 
* we would also like to share this on CZI cluster and beyond.

**Comment by ziw-liu** [comment]
Share environment is live. See [wiki page](https://github.com/czbiohub-sf/napari-iohub/wiki/Shared-environment-on-HPC-(Bruno)). I'm also in conversation with @JoOkuma to consolidate our efforts.
